controllers/endpointconfig/endpointconfig_controller.go

```diff
@@ -150,7 +149,7 @@ func (r *EndpointConfigReconciler) reconcileEndpointConfig(ctx reconcileRequestC
 	}
 
 	// Get SageMaker endpointConfig description.
-	if ctx.EndpointConfigDescription, err = ctx.SageMakerClient.DescribeEndpointConfig(ctx, generateEndpointConfigName(ctx.EndpointConfig)); err != nil {
+	if ctx.EndpointConfigDescription, err = ctx.SageMakerClient.DescribeEndpointConfig(ctx, GetGeneratedResourceName(ctx.EndpointConfig.ObjectMeta.GetUID(), ctx.EndpointConfig.ObjectMeta.GetName(), 63)); err != nil {
```

Comment:
Would you be able to create constants for holding the name lengths for each resource? I've included this in the TrainingJob refactor, as an example.